openmdao/api.py

```diff
@@ -40,10 +39,8 @@
 from openmdao.solvers.run_once import RunOnce
 from openmdao.solvers.scipy_gmres import ScipyGMRES
 from openmdao.solvers.solver_base import LinearSolver, NonLinearSolver
-try:
+if under_mpirun():
     from openmdao.solvers.petsc_ksp import PetscKSP
```

Comment:
Would this prevent you from using PetscKSP when not under MPI? When I am comparing calculations, I like to use the same solvers.